Index: src/java/org/liquibase/grails/GrailsXmlWriter.java
===================================================================
--- src/java/org/liquibase/grails/GrailsXmlWriter.java	(revision 47353)
+++ src/java/org/liquibase/grails/GrailsXmlWriter.java	(working copy)
@@ -2,8 +2,8 @@
 
 import liquibase.xml.XmlWriter;
 import org.w3c.dom.Document;
-import org.apache.xml.serialize.XMLSerializer;
-import org.apache.xml.serialize.OutputFormat;
+import com.sun.org.apache.xml.internal.serialize.XMLSerializer;
+import com.sun.org.apache.xml.internal.serialize.OutputFormat;
 
 import java.io.OutputStream;
 import java.io.IOException;
Index: src/java/org/liquibase/grails/GrailsFileOpener.java
===================================================================
--- src/java/org/liquibase/grails/GrailsFileOpener.java	(revision 47352)
+++ src/java/org/liquibase/grails/GrailsFileOpener.java	(working copy)
@@ -1,52 +0,0 @@
-package org.liquibase.grails;
-
-import liquibase.FileOpener;
-import liquibase.ClassLoaderFileOpener;
-
-import java.io.InputStream;
-import java.io.IOException;
-import java.net.URL;
-import java.util.Enumeration;
-
-import org.codehaus.groovy.grails.commons.ApplicationHolder;
-
-public class GrailsFileOpener implements FileOpener {
-    
-    private FileOpener fileOpener;
-
-    public GrailsFileOpener() {
-        if (ApplicationHolder.getApplication().isWarDeployed()) {
-            fileOpener = new ClassLoaderFileOpener();
-        } else {
-            fileOpener = new DevFileOpener();
-        }
-    }
-
-    public InputStream getResourceAsStream(String file) throws IOException {
-        return fileOpener.getResourceAsStream(file);
-    }
-
-    public Enumeration<URL> getResources(String packageName) throws IOException {
-        return fileOpener.getResources(packageName);
-    }
-
-    public ClassLoader toClassLoader() {
-        return fileOpener.toClassLoader();
-    }
-
-    private static class DevFileOpener implements FileOpener {
-
-        public InputStream getResourceAsStream(String file) throws IOException {
-            return getClass().getClassLoader().getResourceAsStream("grails-app/migrations/"+file);
-        }
-
-        public Enumeration<URL> getResources(String packageName) throws IOException {
-            return getClass().getClassLoader().getResources("grails-app/migrations/"+packageName);
-        }
-
-        public ClassLoader toClassLoader() {
-            return getClass().getClassLoader();
-        }
-
-    }
-}
Index: src/groovy/org/liquibase/grails/GrailsFileOpener.groovy
===================================================================
--- src/groovy/org/liquibase/grails/GrailsFileOpener.groovy	(revision 0)
+++ src/groovy/org/liquibase/grails/GrailsFileOpener.groovy	(revision 0)
@@ -0,0 +1,58 @@
+package org.liquibase.grails;
+
+import liquibase.FileOpener;
+import liquibase.ClassLoaderFileOpener;
+
+import grails.util.GrailsUtil;
+import java.io.*;
+import java.net.URL;
+import java.util.Enumeration;
+
+import org.codehaus.groovy.grails.commons.ApplicationHolder;
+
+class GrailsFileOpener implements FileOpener {
+    
+    def fileOpener;
+
+    public GrailsFileOpener() {
+        if (ApplicationHolder.getApplication().isWarDeployed()) {
+            fileOpener = new ClassLoaderFileOpener();
+        } else {
+            fileOpener = new DevFileOpener();
+        }
+    }
+
+    InputStream getResourceAsStream(String file) throws IOException {
+        fileOpener.getResourceAsStream(file);
+    }
+
+    Enumeration<URL> getResources(String packageName) throws IOException {
+        fileOpener.getResources(packageName);
+    }
+
+    ClassLoader toClassLoader() {
+        fileOpener.toClassLoader();
+    }
+
+}
+
+class DevFileOpener implements FileOpener {
+
+    InputStream getResourceAsStream(String file) throws IOException {
+        if (GrailsUtil.grailsVersion.startsWith('1.0')) {
+            return getClass().getClassLoader().getResourceAsStream("grails-app/migrations/"+file);
+        } else {
+            return new FileInputStream(new File(grails.util.BuildSettingsHolder.settings.baseDir.path + "/grails-app/migrations/" + file));
+        }
+    }
+
+    Enumeration<URL> getResources(String packageName) throws IOException {
+        getClass().classLoader.getResources("grails-app/migrations/"+packageName);
+    }
+
+    public ClassLoader toClassLoader() {
+        getClass().classLoader
+    }
+
+}
+
Index: plugin.xml
===================================================================
--- plugin.xml	(revision 47352)
+++ plugin.xml	(working copy)
@@ -1,7 +1,7 @@
-<plugin name='liquibase' version='1.9.0.0'>
-  <author>Nathan Voxland</author>
-  <authorEmail>nathan@voxland.net</authorEmail>
-  <title>LiquiBase Database Refactoring for Grails</title>
+<plugin name='liquibase' version='1.9.0.0'>
+  <author>Nathan Voxland</author>
+  <authorEmail>nathan@voxland.net</authorEmail>
+  <title>LiquiBase Database Refactoring for Grails</title>
   <description>Integrates LiquiBase into Grails.  LiquiBase is a database change tracking tool similar to ActiveRecord:Migration.
 Major functionality includes:
 - 34 Refactorings
@@ -20,9 +20,13 @@
 - DBMS Check, user check, and SQL check preconditions
 - Can split change log into multiple files for easier management
 - Support for 10 database systems
-</description>
-  <documentation>http://www.liquibase.org/manual/grails</documentation>
-  <resources>
-    <resource>DataSource</resource>
-  </resources>
+</description>
+  <documentation>http://www.liquibase.org/manual/grails</documentation>
+  <resources>
+    <resource>DataSource</resource>
+    <resource>UrlMappings</resource>
+  </resources>
+  <dependencies>
+    <plugin name='dataSource' version='* &gt; 1.0' />
+  </dependencies>
 </plugin>
\ No newline at end of file
Index: application.properties
===================================================================
--- application.properties	(revision 47352)
+++ application.properties	(working copy)
@@ -1,4 +1,5 @@
-#Do not edit app.grails.* properties, they may change automatically. DO NOT put application configuration in here, it is not the right place!
-#Fri Apr 18 14:20:54 CDT 2008
-app.grails.version=1.0.2
-app.name=liquibase
+#utf-8
+#Thu Feb 05 09:27:59 CST 2009
+app.grails.version=1.1-SNAPSHOT
+plugins.hibernate=1.1-SNAPSHOT
+app.name=liquibase
